README.md

```diff
@@ -1,5 +1,57 @@
 # The Interop Project
 
-Welcome to the Interop project, an ongoing effort to make the web more interoperable in key areas, prioritized by web developer and user needs. This is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+Welcome to the Interop Project, an ongoing effort to make the web more interoperable in key areas, prioritized by web developer and user needs. This is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform, and worked on by the [Interop Team](./supporters.md).
```

Comment:
I would suggest putting "users" before "web developers" everywhere. Ultimately helping developers is only important insofar as it ultimately benefits users. Generally their needs are aligned (although there may be cases e.g. when features have privacy concerns, where they are not), but I don't think we should imply that developers' needs outweigh those of end users.